CGBEmu/source/Player.cpp

```diff
@@ -1,24 +1,28 @@
+#include "Player.h"
```

Comment:
Before this PR, you had both the header interface declaration and the definition of the class in the same file. I removed the interface declaration and added it to Player.h instead.

To associate this implementation with the interface declataion we use the preprocessor directive #include "Player.h".